dropwizard-example/example.yml

```diff
@@ -24,6 +24,16 @@ database:
 #    type: http
 #    port: 8080
 
+# run on a single port and server.connector.inheritChannel=true if your application launch by the "Server::Starter"
```

Comment:
I believe it would be better to document this option in the [configuration reference](https://github.com/dropwizard/dropwizard/blob/master/docs/source/manual/configuration.rst) instead of the example configuration file. We prefer to keep it simple for users who start with Dropwizard.